sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -109,4 +109,21 @@ task validatesRunnerStreamingTests {
       args '-c', ". ${envdir}/bin/activate && ${runScriptsDir}/run_integration_test.sh $cmdArgs"
     }
   }
-}
\ No newline at end of file
+}
+
+task runPerformanceTest {
+    dependsOn 'installGcpTest'
+    dependsOn ':sdks:python:sdist'
+
+    def test = project.findProperty('test')
+    def testOpts = project.findProperty('test-pipeline-options')
+    testOpts += " --sdk_location=${files(configurations.distTarBall.files).singleFile}"
+
+  doLast {
+    exec {
+      workingDir "${project.rootDir}/sdks/python"
+      executable 'sh'
+      args '-c', ". ${envdir}/bin/activate && ${envdir}/bin/python setup.py nosetests --tests=${test}  --test-pipeline-options=\"${testOpts}\" --ignore-files \'.*py3\\d?\\.py\$\'"
```

Comment:
Please make sure these tests export the xml logs that can be inspected in Jenkins in case of test failure:

Relevant bits are: 
1. https://github.com/apache/beam/blob/03d99dfa359f44a29a772fcc8ec8b0a237cab113/.test-infra/jenkins/job_PostCommit_Python37.groovy#L32
2. https://github.com/apache/beam/blob/03d99dfa359f44a29a772fcc8ec8b0a237cab113/sdks/python/scripts/run_integration_test.sh#L276

cc: @udim who may have additional feedback on this. Udi, would it make sense to use [pytest](https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-RunningTestsusingpytest) here instead of `nose`?

Reply:
I don't believe we have `--test-pipeline-options` support yet in pytest, so nose is the solution for now.

Reply:
Done